pkg/service/user.go

```diff
@@ -1079,3 +1081,10 @@ func (s *userService) getUserLastLogin(ctx context.Context, userId uuid.UUID) (s
 	}
 	return lastLogin, nil
 }
+
+func (s *userService) AuditLogWebhook(ctx context.Context, req *userrpcv3.AuditWebhook) (*userrpcv3.AuditWebhook, error) {
+
+	UserLoginAuditEvent(ctx, s.al, "login", "username")
```

Comment:
I suppose this was suppose to be the actual user name and not string "username"

Reply:
This has to come in the request body from kratos, just added a placeholder string for now